test/unit/presenters/formats/local_transaction_presenter_test.rb

```diff
@@ -138,5 +147,38 @@ def result
       ]
       assert_equal expected, result[:routes]
     end
+
+    context "[:scotland_availability]" do
+      should "not present the data if local_authority_service was selected" do
+        edition.scotland_availability = { type: "local_authority_service", alternative_url: "" }
+        edition.save!(validate: false)
+
+        expected = nil
+        assert_equal expected, result[:details][:scotland_availability]
+      end
+
+      should "present the type data if unavailable was selected" do
+        edition.scotland_availability = FactoryBot.build(
+          :devolved_administration_availability,
+          type: "unavailable",
+        )
```

Comment:
As there is already a model we can simplify this a little:

```suggestion
        edition.scotland_availability.type = "unavailable"
```

The next test can also be simplified a little.

Reply:
Thanks, I updated.